client/azure/azure.go

```diff
@@ -0,0 +1,132 @@
+package azure
```

Comment:
IMO there is nothing like a client package in an operator. For me these are all service packages that provide business logic to the operator. The services might use clients to connect to external dependencies via some API. I saw this already in the AWS operator and I still do not like it.